test/UnitTestHelpers.hs

```diff
@@ -7,13 +7,16 @@ module UnitTestHelpers
   , stubLevel
   , stubState
   , stubCliState
+  , stubItem
   , testActor
   , testActorId
   , testActorWithItem
   , testCliStateWithItem
   , testFactionId
   , testItemId
-  , testItemKind
+  , emptyMultiGroupItem
```

Comment:
I wouldn't re-export. Just add the original import where needed. Unless there's a good reason not to.